js/angular/app/scripts/modules/userdata/change-timezone-controller.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+angular.module('transitIndicators')
+.controller('OTIUserdataChangeTimezoneController',
+            ['config', '$modalInstance', '$scope', '$state', '$cookieStore', 'OTILocalization', 'timezones',
+            function (config, $modalInstance, $scope, $state, $cookieStore, OTILocalization, timezones) {
+
+    $scope.selectTimezone = function(timezone) {
+        OTILocalization.setTimeZone(timezone.zone).then(function(response) {
+            $modalInstance.close();
```

Comment:
Yes, there should be no other way out of the modal. Ok, didn't know about `backdrop`.